### PR TITLE
fix: genesis import/export for csr module

### PR DESCRIPTION
**Pull request by poorphd**
export/import genesis for csr module does not work due to the missing states in:
[GenesisState](https://github.com/Canto-Network/Canto/blob/main/proto/canto/csr/v1/genesis.proto#L10-L13)
[InitGenesis](https://github.com/Canto-Network/Canto/blob/main/x/csr/genesis.go#L10-L20)
[ExportGenesis](https://github.com/Canto-Network/Canto/blob/main/x/csr/genesis.go#L22-L28)

This patch fixes genesis import/export

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] targeted the correct branch (see [PR Targeting](https://github.com/canto-network/canto/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [x] included the necessary unit and integration tests
- [x] reviewed "Files changed" and left comments if necessary
 
### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: x/csr/genesis.go [hunk 1]
```diff
@@ -23,6 +32,18 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, accountKeeper authkeeper.Acco
 func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
 	genesis := types.DefaultGenesis()
 	genesis.Params = k.GetParams(ctx)
+	csrs := k.GetAllCSRs(ctx)
+
+	if len(csrs) == 0 {
+		genesis.Csrs = []types.CSR{}
+	} else {
+		genesis.Csrs = csrs
+	}
```
Comment (dudong2) [inline x/csr/genesis.go:41]:
```suggestion
	genesis.Csrs := k.GetAllCSRs(ctx)
```

Comment (dudong2) [reply x/csr/genesis.go:41]: may it works, i think...

Comment (dudong2) [reply x/csr/genesis.go:41]: Not work,,,,,,

**Review by dudong2 — APPROVED** [review]
LGTM

**Review by zsystm — APPROVED** [review]
LGTM

**Pull request merged**